CondTools/HLT/src/AlCaRecoTriggerBitsRcdUpdate.cc

```diff
@@ -179,28 +171,17 @@ bool AlCaRecoTriggerBitsRcdUpdate::addTriggerLists(const std::vector<edm::Parame
 }
 
 ///////////////////////////////////////////////////////////////////////
-void AlCaRecoTriggerBitsRcdUpdate::writeBitsToDB(AlCaRecoTriggerBits *bitsToWrite) const {
+void AlCaRecoTriggerBitsRcdUpdate::writeBitsToDB(const std::unique_ptr<AlCaRecoTriggerBits> &bitsToWrite) const {
   edm::LogInfo("") << "Uploading to the database...";
 
   edm::Service<cond::service::PoolDBOutputService> poolDbService;
   if (!poolDbService.isAvailable()) {
     throw cms::Exception("NotAvailable") << "PoolDBOutputService not available.\n";
   }
 
-  // ownership of bitsToWrite transferred
   // FIXME: Have to check that timetype is run number! How?
   const std::string recordName("AlCaRecoTriggerBitsRcd");
```

Comment:
```suggestion
```

Reply:
Good point, I will remove this and use directly the string in the function in my next commit